tests/PersistentArrayOfStringsTest.php

```diff
@@ -46,7 +46,7 @@ public function testTypecasting()
             'array' => ['foo' => 'bar', 'int' => 123, 'rows' => ['a', 'b']],
             'object' => (object) ['foo' => 'bar', 'int' => 123, 'rows' => ['a', 'b']],
         ]);
-        $m->saveAndUnload();
+        (clone $m)->saveAndUnload();
```

Comment:
$m is not loaded here so why clone ?

Reply:
I kept `saveAndUnload` (`save` whould be enough), but we need clone because `save` makes model entity as well (which is consistent, as model is associated with DB, no matter if from laod or save)